geowebcache/core/src/main/java/org/geowebcache/GeoWebCacheExtensions.java

```diff
@@ -349,66 +349,6 @@ public static String getProperty(String propertyName) {
      * @return The property value, or null if not found
      */
     public static String getProperty(String propertyName, ApplicationContext context) {
-        if (context instanceof WebApplicationContext) {
-            return getProperty(propertyName, ((WebApplicationContext) context).getServletContext());
-        } else {
-            return getProperty(propertyName, (ServletContext) null);
-        }
-    }
-
-    /**
-     * Looks up for a named string property into the following contexts (in order):
-     *
-     * <ul>
-     *   <li>System Property
-     *   <li>web.xml init parameters
-     *   <li>Environment variable
-     * </ul>
-     *
-     * and returns the first non null, non empty value found.
-     *
-     * @param propertyName The property name to be searched
-     * @param context The servlet context used to look into web.xml (may be null)
-     * @return The property value, or null if not found
-     */
-    public static String getProperty(String propertyName, ServletContext context) {
```

Comment:
Note this one is safe to remove as it was only called from a test case. Not used in geoserver either.

Reply:
I don't want to rule out the possibility that it's being used in some custom extension or modification somewhere so I'd be happier if it were deprecated for a release or two first before removing it entirely

Reply:
You got it https://github.com/GeoWebCache/geowebcache/pull/989/files#diff-88bfaa979935d2ccfabe9c9cf93d6628892c241905bc012c94192588d6f7621fR371-R374